src/Stratis.SmartContracts.CLR/Stratis.SmartContracts.CLR.csproj

```diff
@@ -17,6 +17,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Stratis.SCL\Stratis.SCL.csproj" />
```

Comment:
This reference should be removed as `Stratis.SCL` should only be used by smart contracts.

Reply:
Unused reference, I have removed it

Reply:
The reference still exits implicitly.